Benchmarks/FourParameters.cs

```diff
@@ -0,0 +1,64 @@
+using BenchmarkDotNet.Attributes;
+using BenchmarkMockNet.PCLMock;
+using FakeItEasy;
+using Moq;
+using NSubstitute;
+using Rocks;
+using System;
+
+namespace BenchmarkMockNet.Benchmarks
+{
+   public class FourParameters : MockingBenchmark
+   {
+      private readonly string _a = "a";
+      private readonly Guid _b = Guid.NewGuid();
+      private readonly string _c = "c";
+      private readonly Guid _d = Guid.NewGuid();
+
+      [Benchmark(Baseline = true)]
+      public override void Stub()
+      {
+         var stub = new ThingStub();
+         stub.FourParameters(_a, _b, _c, _d);
+      }
+
+      [Benchmark]
+      public override void FakeItEasy()
+      {
+         var fake = A.Fake<IThingy>();
+         A.CallTo(() => fake.FourParameters(_a, _b, _c, _d));
```

Comment:
FakeItEasy, Moq, and NSubstitute all handle method calls that return default values without the need to explicitly set them up (see `ReturnEmpty`), so only PCLMock and Rocks need these lines I believe.

Reply:
I was trying to follow what the other test code was doing. Rocks has two types of mocks: "create" (mocks with expectations), and "make" (mocks without). So, it's not quite fair to use a "make" for these tests because they barely doing anything.

I **can** make the tests not do any setup/expectations for these frameworks, but I would think this should be consistent as well. Not sure exactly how to do that....I guess with my update, I'll remove these setups, but the "fairness" question should be taken into consideration.